demo_data/data.sql

```diff
@@ -2941,13 +2941,13 @@ INSERT INTO %PREFIX%_user_data (usd_id, usd_usr_id, usd_usf_id, usd_value) VALUE
 -- Data for table adm_user_log
 --
 
-INSERT INTO %PREFIX%_user_log (usl_id, usl_usr_id, usl_usf_id, usl_value_new, usl_value_old, usl_usr_id_create, usl_timestamp_create, usl_comment) VALUES
```

Comment:
We shouldn't delete these lines now because we want to simulate the update process from user_log to log_changes

Reply:
As I said, I misunderstood the demo data fundamentally... reverted. And the update process should be implemented.